WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -1806,6 +1808,10 @@ private void showPrepublishingNudgeBottomSheet() {
         }
     }
 
+    @Override public void onPublishButtonClicked() {
+        uploadPost(true);
```

Comment:
This logic is improved here https://github.com/wordpress-mobile/WordPress-Android/pull/11824
since not all actions that trigger the bottom sheet should set `publishPost` as true. I created another PR since it's outside the scope of this one.